### ISSUE TITLE
Terms of Service

### ISSUE THREAD

**Issue by robhawkes** [polygon-city/building-database#42]
As mentioned in #37, we're going to need some sort of ToS to point people toward in regards to expected behaviour, terms of use, etc.

I don't know exactly what that needs to consist of but we can use this issue to track it.

**Comment by meetar** [comment]
Need to consider license – allowing commercial use may restrict possible data sources such as the UK lidar which requires not-for-profit use

**Comment by robhawkes** [comment]
It's possible that even a non-profit license CC or otherwise would still restrict the use of data like UK lidar. It's worth looking into though.

**Comment by robhawkes** [comment]
Part of #53

**Comment by robhawkes** [comment]
What's the "it" milestone? :D

**Comment by robhawkes** [comment]
It may be worth moving this into another thread, however I'm currently looking into open-data licensing and how the options compare. [I found something interesting](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/) which is that anything released under the Open Government License v3 is immediately compatible with both Creative Commons 4.0 Attribution and Open Data Commons Attribution licenses. This is good because it covers basically anything public-sector in the UK that's been released in the past few years!

Unfortunately, a lot of data (eg. the UK lidar stuff) is still released under Crown Copyright and other random licenses which I've not worked out the compatibility with just yet. I doubt we'd be in luck there as the Environment Agency (who own the lidar data) are in the business of selling it so even the non-profit use is highly restricted.

**Comment by robhawkes** [comment]
For reference the US government seem to release most (all?) of their data as public domain, specifically [Creative Commons CC0 Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/). So that's good.

**Comment by meetar** [comment]
(…I have no idea what the "it" milestone is or how I added it.)

**Comment by robhawkes** [comment]
A blank terms page has been added with d058af2 and can be filled out once we have the terms content.

**Comment by robhawkes** [comment]
I found [this comparison of open data licenses and their compatibility](https://github.com/theodi/open-data-licensing/blob/master/guides/licence-compatibility.md) quite interesting – it looks like CC 4.0 has removed a lot of (maybe all of) the need for ODbL.

**Comment by meetar** [comment]
This is excellent, and something I've been wanting to see.

**Comment by robhawkes** [comment]
Definitely. I saw an (old) article by OSM during my research which stated that they foresee CC reaching compatibility with ODbL in the future. Perhaps that time has arrived with 4.0.
